content/applications/general/email_communication/email_servers.rst

```diff
@@ -99,29 +99,6 @@ that Odoo is trying to redistribute would get rejected by some recipients' email
 
 To avoid that problem, make sure all emails use a "From" address from the same authorized domain.
```

Comment:
```suggestion
To avoid this problem, Odoo has a notifications configuration. The following system parameters should be set in order for notifications to be sent out from Odoo: 

`mail.default.from_filter` : Value is a domain. This parameter is used to redistribute responses effectively; if a ‘From’ email address's domain doesn’t match this parameter, it is rewritten and [mail.default.from] @ [mail.catchall.domain] is used instead.

`mail.default.from` : Value is “notifications” by default. Only used when mail.default.from_filter does not match the author’s domain.

You access the **System Parameters** in :ref:`developer mode <developer-mode>` in the :menuselection:`Settings -->
Technical --> Parameters --> System Parameters` menu.
```

Reply:
@bouvyd  Hi Damien, when you have a moment can you review this doc? Thanks and happy holidays. 👍

Reply:
Hello @tiku-odoo 
I'm having a quick look, it's already explained in the email_domain file, should it be put together ? With a redirection for the other page ?

Reply:
Yes, already explained here:
content/applications/general/email_communication/email_domain.rst